sky/resources.py

```diff
@@ -5,9 +5,9 @@
 
```

Comment:
Should we add the ports check back that is reverted in #3175

Reply:
It is still possible to cause some bugs using the current version of `get_enabled_clouds`: consider there is a public cloud available in the local database (on the controller), but the cloud does not support ports, and the user just enabled a new cloud on the local laptop; in this case the `get_enabled_clouds` will not trigger a refresh and still causing the original problem. I'm thinking to separate it to a new PR and force refresh before checking ports.

Reply:
In that case, it will raise the error on the local machine directly, instead of only raising on the serve controller, right?

Reply:
Oh that makes sense! SG

Reply:
Will add it after this round of smoke test finished